.github/workflows/tests.yml

```diff
@@ -48,26 +48,17 @@ jobs:
       matrix:
         include:
           - name-suffix: "(Minimum Versions)"
-            os: ubuntu-20.04
-            python-version: '3.10'
+            os: ubuntu-22.04
+            python-version: '3.11'
```

Comment:
Minimum version should still test Python 3.10 for the 3.10 branch.

Reply:
I think minver should be good enough to catch py310 incompatible code.